src/pages/RepoPage/FlagsTab/FlagsTab.spec.jsx

```diff
@@ -84,21 +108,43 @@ describe('Flags Tab', () => {
           ctx.status(200),
           ctx.data({ owner: { plan: { tierName: TierNames.PRO } } })
         )
+      }),
+      graphql.query('GetRepoSettingsTeam', (req, res, ctx) => {
+        return res(ctx.status(200), ctx.data(mockRepoSettings(isPrivate)))
       })
     )
   }
 
   describe('when user has a team tier', () => {
-    beforeEach(() => {
-      setup({ tierValue: TierNames.TEAM })
+    describe('the repo is public', () => {
+      it('renders the flags tab', async () => {
+        setup({
+          tierValue: TierNames.TEAM,
+          isPrivate: false,
```

Comment:
mega nit: do you need this?

Reply:
So because this is testing that the repo is not private I like to be entirely explicit in the setup for things that are related to the test. This means that you don't have to go up and check the setup function and the arguments default value to see if it aligns with what's being tested.

If one day for whatever reason we switch the default value then this test will also still continue to work properly.